feat: GitHub templates (#3)
Soheab/dpy-missing-features: .github/PULL_REQUEST_TEMPLATE.md

## Brief Description

<!-- Briefly explain your PR -->

## Detailed Description

<!-- Explain with as much detail as possible your PR -->

## Change Log

> <!-- Here add the changes you've made as unordered list -->
> <!-- WHEN MAKING A PR, REMOVE THIS LINE
> - Change log 1
> - Change log 2
> WHEN MAKING A PR, REMOVE THIS LINE -->

Reviewer: ```suggestion
<!-- List the changes you made, does not have to be in any particular order -->
- Example log 1
- Example log 2
```
too much imo
